demo/src/main/java/com/jira/demo/repository/EmployeeRepo.java

```diff
@@ -1,10 +1,9 @@
 package com.jira.demo.repository;
 
 
-
 import com.jira.demo.model.Employee;
 import org.springframework.data.jpa.repository.JpaRepository;
 
-public interface EmployeeRepo extends JpaRepository <Employee, Long>{
+public interface EmployeeRepo extends JpaRepository<Employee, Long> {
```

Comment:
Don't forget to format your code with combination of `Ctrl + Alt + L` to keep your code clean and consistent